api/client/client.go

```diff
@@ -37,6 +41,8 @@ func NewClient(target string, opts ...grpc.DialOption) (*Client, error) {
 		Wallet:     &Wallet{client: walletPb.NewAPIClient(conn)},
 		Reputation: &Reputation{client: reputationPb.NewAPIClient(conn)},
 		Ffs:        &ffs{client: ffsPb.NewAPIClient(conn)},
+		Health:     &health{client: healthRpc.NewHealthClient(conn)},
+		Net:        &net{client: netRpc.NewNetClient(conn)},
```

Comment:
Planning on making the rest of the client modules non-exported like this so they can only be accessed through the top level `Client`